demo/.gitignore

```diff
@@ -1 +1,4 @@
 config.json
+
+bin
+generated
```

Comment:
We should be committing generated code to the demo, no? This is generally considered best practice, and we do it with all the gqlgen subgraphs.

Reply:
If you want to ensure it is identical to committed version in CI, I recommend something like this:

```bash
#!/bin/bash

set -euo pipefail

task alyx:gen -f

status=$(git status --porcelain)
statusv=$(git diff)

if [ -n "$status" ]; then
	echo "you need to generate and commit the changes"
	echo "$statusv"
	exit 1
fi
```